src/ResourceManager/MachineLearning/Commands.MachineLearning.Test/ScenarioTests/BaseTestController.cs

```diff
@@ -152,10 +152,10 @@ private void SetupManagementClients(MockContext context)
             HttpClientHelperFactory.Instance = new TestHttpClientHelperFactory(credentials);
 
             helper.SetupManagementClients(
-                this.resourceManagementClient, 
-                subscriptionClient, 
-                this.serviceClient, 
-                authManagementClient, 
+                this.resourceManagementClient,
```

Comment:
why are these changes included?

Reply:
I have an extension in VS that trims trailing whitespace (😀), so outside of the change in instantiating the ResourceManagementClient, everything else in this file is whitespace trimming